README.md

```diff
@@ -1,13 +1,389 @@
+              	"headers": 
+         		{	
+                		"content-type": "application/JSON"
+              		}
+         				.then (res => res.json())
+         				.then ((data) => {
+         					//do stuff with your data
+         				})
+       	 })
+	
+##### Success Response
+
+	{
+		"success": true,
+		"message": "Dish successfully added to order",
+		"status": 200,
+	}
+
+##### Error Response
+
+	{
+		"success": false,
+		"message": "Item not found so cannot add to order",
+		"status": 400
+	}
+
+
+### Edit quantities of dishes in orders
+
+`PUT /orders/editQuantity`
+
+##### Data Params
+
+	{
+           "_id": "60c73afb0b5f5c23d4a61688"
+           "orderItems": [{"menuItemId": "60c73afb0b5f5c23d4a61688" , "quantity": 1}]
+        }
+
+##### Sample Call
+
+	 fetch('http://localhost:3000/orders', {
+                  "method": PUT,
+                  "body": JSON.stringify(/* your data goes here */),
+                  "headers": 
+           	{  
+                   "content-type": "application/JSON"
+               	}
+                 	.then (res => res.json())
+                 	.then ((data) => {
+                  		  //do stuff with your data
+                })
+        })
+
+##### Success Response
+
+	{
+		"success": true,
+		"message": "Your dish quantity was successfully edited.",
+		"status": 200
+	}
+
+##### Error Response
+	
+If the DB fails to update the dish quantity:
+	
+	{
+		"success": false,
+		"message": "Dish quantity was not updated",
+		"status": 400
+	}
+	
+If the dish ID is incorrect:
+	
+	{
+		"success": false,
+		"message": "There is no dish found with that ID.",
+		"status": 404
+	}
+
+
+#### Removing an item (in any quantity) from an order
+
+`PUT /orders/removeDish`
+
+This endpoint allows you to remove an item entirely from an order.
+
+##### Data Params
+
+	 {
+           "_id": "60c73afb0b5f5c23d4a61688"
+           "orderItems": [{"menuItemId": "60c73afb0b5f5c23d4a61689", "quantity": 1}]
+        }
+
+##### Sample Call
+
+	fetch('http://localhost:3000/orders', {
+        	"method": PUT,
+                "body": JSON.stringify(/* your data goes here */),
+                "headers": 
+           	{  
+                   "content-type": "application/JSON"
+               	}
+                 	.then (res => res.json())
+                	 .then ((data) => {
+                    		//do stuff with your data
+                	 })
+       		 })
+
+##### Success Response
+	{
+		"success": true,
+		"message": "Dish successfully deleted from order",
+		"status": 200,
+	}
+
+##### Error Response
+	
+	{
+		"success": false,
+		"message": "Item not found so cannot be deleted from order",
+		"status": 404
+	}
+
+### Submit final order
+
+`PUT /orders/submitOrder`
+
+This endpoint will send your order to the restaurant and cannot be taken back. 
+
+##### Data Params
+
+	 {
+           "_id": "60c73afb0b5f5c23d4a61688"
+           "orderItems": [{"menuItemId": "60c73afb0b5f5c23d4a61689", "quantity": 1}]
```

Comment:
I think the user would only need to provide their orderId and not the items as part of the request. This is because the order is already in our database.